### ISSUE TITLE
User profile in Dark Mode

### ISSUE THREAD

**Issue by Caribsky** [oznu/homebridge-config-ui-x#363]
**Describe The Bug:**

The Username field is not so good visible in Dark Mode 

![Снимок экрана 2019-08-16 в 1 25 02](https://user-images.githubusercontent.com/37780308/63129879-cb339d80-bfc9-11e9-9093-10da6ca498f6.png)

**To Reproduce:**

**Expected behavior:**

**Logs:**

```
Show the Homebridge / Homebridge Config UI X logs here.
```

**Homebridge Config:**

```json
Show your homebridge config.json here
```

**Screenshots:**

**Environment:**

* **Node.js Version**: 
* **NPM Version**: 
* **Homebridge Version**: v4.6.0  
* **Homebridge Config UI X Version**:
* **Operating System**: Raspbian / Ubuntu / Debian / Windows / macOS / Docker
* **Process Supervisor**: Docker / Systemd / init.d / pm2 / launchctl / other / none

**Comment by grzegorz914** [comment]
Same for me

**Comment by oznu** [comment]
This should be fixed in 4.6.1... if you want to try the beta release:

```
sudo npm install -g --unsafe-perm homebridge-config-ui-x@next
```
